pkg/cloudprovider/providers/gce/gce_loadbalancer_external.go

```diff
@@ -72,6 +69,21 @@ func (gce *GCECloud) ensureExternalLoadBalancer(clusterName, clusterID string, a
 	glog.V(2).Infof("EnsureLoadBalancer(%v, %v, %v, %v, %v, %v, %v)",
 		loadBalancerName, gce.region, requestedIP, portStr, hostNames, serviceName, apiService.Annotations)
 
+	lbRefStr := fmt.Sprintf("%v(%v)", loadBalancerName, serviceName)
+	// Check the current and the desired network tiers. If they do not match,
+	// tear down the existing resources -- delete the forwarding rule, and
+	// release the IP.
+	netTier, err := gce.getServiceNetworkTier(apiService)
+	if err != nil {
+		glog.Errorf("EnsureLoadBalancer(%s): failed to get the desired network tier: %v", lbRefStr, err)
+		return nil, err
+	}
+	glog.V(2).Infof("EnsureLoadBalancer(%s): desired network tier %q ", lbRefStr, netTier)
```

Comment:
User that doesn't enable network tier may be seeing this "network tier" log and they may think it is mistakenly enabled?

Reply:
Bumped it to V(4) so that it doesn't show up in GKE clusters.

Reply:
Sorry for switching side...Actually I'd prefer to keep this in V(2) for better debuggability in case things go wrong in production. (I'd like to feature gate it, or maybe it just doesn't matter)